notebooks/SMAP/2.0 Read and Plot SMAP data.ipynb

```diff
@@ -0,0 +1,549 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    print('By using the command h5py.File() a filehandle is returned:')
</pre><p>Suggest delete - can be explained in the preceding markdown cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nsidc/NSIDC-Data-Tutorials/pull/57/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>